samples/sobel/sobel_main.py

```diff
@@ -1,56 +1,231 @@
-from PIL import Image
+"""
+Sobel Edge Detection
+=====================
+
+**Authors**: YoungSeok Na, Alga Peng, Xiangyi Zhao, Mira Kim
+
+In this example, we will demonstrate Sobel Edge Detection algorithm written in
+HeteroCL.
```

Comment:
It should be better if we give more details here. E.g., it is a image processing algorithm. And most importantly, we are demonstrating the HLS report feature which makes programmers' life much easier.